common/v2/features/Dashboard/components/WalletBreakdown/WalletBreakdown.tsx

```diff
@@ -107,13 +107,17 @@ export function WalletBreakdown() {
             toggleShowChart={toggleShowChart}
             totalFiatValue={totalFiatValue}
             fiat={fiat}
+            accounts={accounts}
+            selected={settings.dashboardAccounts}
```

Comment:
I think you forgot to remove this.

Reply:
I can't remove does since `BalancesDetailView` relies on `WalletBreakdownProps` and I need those on `WalletBreakdownView`. Should I create a new PropType for `BalancesDetailView` ?

Reply:
@vilsbole ?